### ISSUE TITLE
Line numbers not appearing since recent Joplin update

### ISSUE THREAD

**Issue by thesunshade** [shantanugoel/joplin-plugin-cm-linenumbers#5]
Since recently updating to Joplin 2.3.5, the line numbers are no longer appearing.

I think it is related to the [information in this notice](https://discourse.joplinapp.org/t/important-plugins-that-use-deprecated-apis-will-no-longer-work/19278).

I see this in the console (although I'm not sure if I copied everything correctly):

```
C:\Program Files\Joplin\resources\app.asar\node_modules\@joplin\lib\Logger.js:188 15:03:35: joplin.plugins: Uncaught exception in plugin "com.shantanugoel.JoplinCMLineNumbersPlugin": Error: "com.shantanugoel.JoplinCMLineNumbersPlugin": No longer supported: joplin.plugins.registerContentScript() is deprecated in favour of joplin.contentScripts.register() (deprecated since version 1.8)
    at Plugin.deprecationNotice (C:\Program Files\Joplin\resources\app.asar\node_modules\@joplin\lib\services\plugins\Plugin.js:117)
    at JoplinPlugins.<anonymous> (C:\Program Files\Joplin\resources\app.asar\node_modules\@joplin\lib\services\plugins\api\JoplinPlugins.js:60)
    at Generator.next (<anonymous>)
    at C:\Program Files\Joplin\resources\app.asar\node_modules\@joplin\lib\services\plugins\api\JoplinPlugins.js:8
    at new Promise (<anonymous>)
    at __awaiter (C:\Program Files\Joplin\resources\app.asar\node_modules\@joplin\lib\services\plugins\api\JoplinPlugins.js:4)
    at JoplinPlugins.registerContentScript (C:\Program Files\Joplin\resources\app.asar\node_modules\@joplin\lib\services\plugins\api\JoplinPlugins.js:59)
    at Object.<anonymous> (C:\Program Files\Joplin\resources\app.asar\node_modules\@joplin\lib\services\plugins\utils\executeSandboxCall.js:50)
    at Generator.next (<anonymous>)
    at C:\Program Files\Joplin\resources\app.asar\node_modules\@joplin\lib\services\plugins\utils\executeSandboxCall.js:8
```

This is a great plugin! I wish that it was part of core, really. Thank you for creating it and I hope it can get working again.

**Comment by thesunshade** [comment]
Did some poking around and it seems that the problem is in line 25 in /api/JoplinPlugins.d.ts

**Comment by shantanugoel** [comment]
Thanks for creating this. This is already fixed by #2 If you are able to build the plugin, please try with the current tip of the code. Otherwise I will try and make a release along with adding it to plugin repository (as asked in #7 )

**Comment by shantanugoel** [comment]
Released here: https://github.com/shantanugoel/joplin-plugin-cm-linenumbers/releases/tag/v2.0.0

**Comment by thesunshade** [comment]
I just installed manually and it works. Thank you so much!

>This is already fixed by #2 

Sorry, I poked around but hadn't seen that.
